### PR TITLE
Model/supply demand

### PR DESCRIPTION
**Pull request by itsjcao**

### CODE REVIEW

File: marketplace/supply-availability/R/get-data.R [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+JOIN greater_lane_volume ON shipments.id = greater_lane_volume.id
+WHERE shipments.falloff_count = 0
+AND shipper_friendly_prefix != 'SCX'
+AND shipments.finished_at > shipments.pickup_driver_end_time
+AND shipments.last_event_name = 'FINISHED'
+AND shipments.last_event_occurred_at BETWEEN '20170801' AND '20171031' 
+"
+df <- get_query(conn, query)
+path <- "supply_availability.feather"
+write_feather(df, path)
```
Comment (mfrasco6) [inline marketplace/supply-availability/R/get-data.R]: So, you can actually do this in one function call with `dshelperfuns::get_data`. You would do `get_data(conn = conn, query = query, write_file_name = path)`. Then, when you want read that file from disk, you would do `get_data(file_name = path)`.

**Comment by itsjcao** [comment]
@mfrasco6 only if you have time!  It was a very long query, maybe you know how to optimize it.

File: marketplace/supply-availability/R/get-data.R (hunk 1 above)
Comment (mfrasco6) [reply marketplace/supply-availability/R/get-data.R]: Actually, I changed this. I created the function `save_data` to save a feather or csv file. And I removed write_file_name from `get_data`

Comment (itsjcao) [reply marketplace/supply-availability/R/get-data.R]: I will look into the new functions and follow the format in the future!

**Pull request merged**